src/__tests__/starWarsData.js

```diff
@@ -122,6 +122,8 @@ function getCharacter(id) {
 
 /**
  * Allows us to query for a character's friends.
+ *
+ * @internal
```

Comment:
@craicoverflow Thanks a lot for PR 👍 
One I didn't mention is that `__tests__` & `__fitures__` are not part of NPM package so all function exported from these folders are private by definition.
So can you please revert all changes under `__tests__` folder?